modules/custom/sql/safe_outposts.sql

```diff
@@ -0,0 +1,16 @@
+-- Relocates dangerous monsters away from outposts
+UPDATE mob_spawn_points SET pos_x =  210.481, pos_y =  -6.946, pos_z =  101.274 WHERE mobname = 'Goblin_Leecher' AND pos_x = 184.673 AND pos_y = -7.710 AND pos_z = 106.691;    -- Valkurm Dunes, Goblin Leecher
```

Comment:
Could you split each of these chunks by Zone, and align them with themselves please, then theres no need for the trailing comments since the mob name is baked in? Otherwise looks good 👍 

```sql
-- Valkurm Dunes
UPDATE mob_spawn_points SET pos_x = 210.481, pos_y = -6.946, pos_z = 101.274 WHERE mobname = 'Goblin_Leecher' AND pos_x = 184.673 AND pos_y = -7.710 AND pos_z = 106.691;

-- Buburimu Peninsula
UPDATE mob_spawn_points SET pos_x = -396.957, pos_y = -32.261, pos_z = 44.893 WHERE mobname = 'Goblin_Shepherd' AND pos_x = -166.429 AND pos_y = 7.910 AND pos_z = -105.478;
UPDATE mob_spawn_points SET pos_x = -411.430, pos_y = -31.429, pos_z =  3.221 WHERE mobname = 'Goblin_Bandit'   AND pos_x = -134.000 AND pos_y = 7.000 AND pos_z =  -96.000;

...
```

Reply:
all set 👍